src/main/scala/viper/gobra/frontend/info/implementation/property/Assignability.scala

```diff
@@ -43,16 +44,22 @@ trait Assignability extends BaseProperty { this: TypeInfoImpl =>
     case (Single(lst), Single(rst)) => (lst, rst) match {
 
         // for go's types according to go's specification (mostly)
+      case (IntT(UntypedConst), r) if isIntegerType(underlyingType(r)) => true
+      case (l, IntT(UntypedConst)) if isIntegerType(underlyingType(l)) => true // not in spec
```

Comment:
The comment is confusing. "Not part of Go spec, but necessary for the definition of comparability".

Reply:
Fixed in 8cb139e